source/server-discovery-and-monitoring/tests/integration/cancel-server-check.yml

```diff
@@ -0,0 +1,96 @@
+# Test SDAM error handling.
+runOn:
+  # General failCommand requirements (this file does not use appName
+  # with failCommand).
+  - minServerVersion: "4.0"
+    topology: ["replicaset"]
+  - minServerVersion: "4.2"
+    topology: ["sharded"]
```

Comment:
Note that this test passes on both 4.4+ and <4.4 servers (both streaming and polling).